### PR TITLE
Adds accessibility guide page

### PR DESCRIPTION
**Pull request by clpetersonucf**
Adds a page to the developer section with guidelines and best practices to consider when making widgets accessible.

### CODE REVIEW

File: _developers/widget-accessibility.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+For example, **Sort It Out** involves an assortment of items cluttered on the screen above a series of folders, which represent categories. Users are tasked with sorting each item into its appropriate category. The relationship between items is fundamentally different for sighted and non-sighted users:
+
+{% include figure.html
+	url="accessibility-example-1.png"
+	alt="Sort It Out player."
+%}
+
+- **Sighted users** see the items as an unordered pile, and the folders as a separate set of elements on the page. There is an implicit understanding that items must be dragged into the corresponding folders to categorize them.
+- **Non-sighted users** percieve the items as an unordered list when described by a screenreader. There is no non-sighted analog to dragging and dropping: rather, screenreader users make use of additional key controls to cycle between potential categories that the item belongs to. When the appropriate category is selected, they perform an additional keyboard input to confirm the choice.
```
Comment (FrenjaminBanklin) [inline _developers/widget-accessibility.md]: perceive

**Review by FrenjaminBanklin — CHANGES_REQUESTED** [review]
I only identified one typo, otherwise this looks great.

**Review by FrenjaminBanklin — APPROVED** [review]
The changes look good.

**Review by cayb0rg — APPROVED** [review]
^^

**Pull request merged**